ykrt/try_passes.py

```diff
@@ -31,11 +31,8 @@ def get_pipeline_config(flag_type, ok_passes, try_passes):
     
     if flag_type == 'lto':
         return PipelineConfig([], ok_passes + try_passes)
-    elif flag_type == 'prelink':
-        return PipelineConfig(ok_passes + try_passes, [])
     else:
-        raise ValueError(f"Unknown flag_type: {flag_type}")
-
+        return PipelineConfig(ok_passes + try_passes, [])
```

Comment:
We can still get rid of the stringly-typed programming here.

If we rename `flag_type` to `is_prelink` which is a bool, then it can reduce to:

```
if prelink:
  ...
else
  ...
```

(don't forget to update callsites)